### ISSUE TITLE
"Applied Accessibility - Improve Chart Accessibility with the figure Element" cannot be completed in Firefox due to issue with <div> tests

### ISSUE THREAD

**Issue by SuperGeniusZeb** [freeCodeCamp/learn#134]
https://learn.freecodecamp.org/responsive-web-design/applied-accessibility/improve-chart-accessibility-with-the-figure-element

This lesson cannot be completed in Firefox because one of the tests ("Your code should not have any `div` tags.") always fails, no matter what the content of the editor is. This issue does not occur in Chromium.

Related:
- #127
- #128
- #129
- #130
- #136

**Comment by Bouncey** [comment]
I cannot reproduce on firefox

**Comment by SuperGeniusZeb** [comment]
I should have clarified, but I was using Firefox Nightly. I recently switched to Firefox Beta, and the issue is still occurring there. I am using Antergos Linux.

I tried disabling various Firefox add-ons, and it turns out that Plasma Integration is the add-on causing the issue:

https://addons.mozilla.org/en-US/firefox/addon/plasma-integration/

Disabling the add-on makes the `<div>` tests immediately start working without even refreshing the lesson page. Likewise, enabling the add-on immediately causes the `<div>` tests to stop working.

I have no idea why this add-on causes an issue. This issue only seems to happen with tests for `<div>`s, not any other HTML element.

The corresponding Chromium extension does not cause any issues with the freeCodeCamp tests. I have no idea if it is because of a difference between Firefox and Chromium, a difference between the Firefox add-on and its matching Chromium extension, both, or neither.

https://chrome.google.com/webstore/detail/plasma-integration/cimiefiiaegbelhefglklhhakcgmhkai

**Comment by raisedadead** [comment]
https://github.com/freeCodeCamp/learn/issues/136#issuecomment-398936869
